assets/sass/components/analytics-4/_googlesitekit-analytics-setup-module.scss

```diff
@@ -46,34 +46,40 @@
 				margin-bottom: 0;
 			}
 
-			.googlesitekit-analytics-enable-enhanced-measurement__already-enabled-label {
-				font-size: $fs-body-sm;
-				letter-spacing: $ls-xs;
-			}
-
-			.googlesitekit-module-settings-group__helper-text {
-				color: $c-surfaces-on-surface-variant;
-				font-size: $fs-body-sm;
-				letter-spacing: $ls-xs;
-				margin: 4px 0;
-			}
-
 			.googlesitekit-plugin-conversion-tracking-notice {
 				color: $c-surfaces-on-surface;
 				font-size: $fs-body-sm;
 				letter-spacing: $ls-xs;
 				margin: 23px 0 0;
 			}
 
-			.googlesitekit-analytics-enable-enhanced-measurement {
+			// Ensure that these styles are only applied within Analytics setup.
+			.googlesitekit-analytics-setup__form {
+
+				.googlesitekit-analytics-enable-enhanced-measurement {
+
+					.googlesitekit-module-settings-group__helper-text {
+						color: $c-surfaces-on-surface-variant;
+						font-size: $fs-body-sm;
+						letter-spacing: $ls-xs;
+						margin: 8px 0 0 42px;
```

Comment:
The vertical margin has been changed from 4px to 8px, specifically for the case when EM is not enabled, to reflect the Figma designs.

<img width="1324" height="336" alt="Image" src="https://github.com/user-attachments/assets/df7b796d-92ec-40d0-83be-1bb7c3ef6c7f" />